.travis.yml

```diff
@@ -12,8 +12,22 @@ before_cache:
 matrix:
   fast_finish: true
   include:
-    - name: Run Rust tests
+    - name: Run Rust tests(linux)
       language: rust
+      os: linux
+      rust: stable
+      cache: cargo
+      script:
+        - cargo build --verbose --all
+        - cargo test --verbose --all
+      env:
+        # Prevention of cache corruption.
+        # See: https://docs.travis-ci.com/user/caching/#caches-and-build-matrices
+        - JOBCACHE=1
```

Comment:
Please choose a different `JOBCACHE`. You now have the same one for linux and osx.

Reply:
@palaviv Thanks for the comment, I've updated job cache

Reply:
@corona10 please take a look at all other jobs as well, we require unique cache values for each job, otherwise jobs will become corrupted.